modules/ztp-acm-adding-images-to-mirror-registry.adoc

```diff
@@ -0,0 +1,45 @@
+// CNF-1500 ZTP - preparing to install ACM
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+[id="ztp-acm-adding-images-to-mirror-registry_{context}"]
+= Adding {op-system} ISO and RootFS images to the disconnected mirror host
+
+Use the disconnected mirror to host the {op-system-first} images you require to provision your DU bare-metal hosts.
+
+.Prerequisites
+
+* Deploy and configure a HTTPD server to host the {op-system} image resources on the network.
+
+.Procedure
+
+. Log in to the mirror host.
+. Download the required {op-system} installation ISO and RootFS images from link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/[mirror.openshift.com], and move them to a folder that is accessible over the network, for example, `/var/www/html`:
++
+[source,terminal]
+----
+$ sudo wget https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest-4.8/rhcos-4.8.0-fc.9-x86_64-live-rootfs.x86_64.img -O /var/www/html/rhcos-4.8.0-fc.9-x86_64-live-rootfs.x86_64.img
```

Comment:
Are you sure these are the right images now and will be the right images for some period going forward? It's a maintenance nightmare to provide `wget` statements for particular versions if the links aren't super stable, so I suggest considering making this guidance less specific. [Here's a topic](https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-iso_installing-bare-metal) where you download ISOs with an approach to consider, at some point.

Reply:
I'll try and get this updated tomorrow.

Reply:
Ok, I've updated this, it's much cleaner now and uses export variables which minimises the hardcoded refererences and makes the procedure more useable.